xbmc/addons/Repository.h

```diff
@@ -70,10 +70,13 @@ namespace ADDON
     explicit CRepositoryUpdateJob(const RepositoryPtr& repo);
     ~CRepositoryUpdateJob() override = default;
     bool DoWork() override;
+    void ShouldInstallAddons(const bool install) { m_shouldinstalladdons = install; };
```

Comment:
Maybe rename to `SetShouldInstallAddons`? Also, for basic types the const is not really required (but it does no harm).